tests/scripts/local_monitoring_tests/rest_apis/test_import_metadata.py

```diff
@@ -0,0 +1,197 @@
+    print("****************************************************\n")
+    tmp_json_file = "/tmp/import_metadata_" + test_name + ".json"
+
+    print("tmp_json_file = ", tmp_json_file)
+
+    form_kruize_url(cluster_type)
+
+    environment = Environment(loader=FileSystemLoader("../json_files/"))
+    template = environment.get_template("import_metadata_template.json")
+    if "null" in test_name:
+        field = test_name.replace("null_", "")
+        json_file = "../json_files/import_metadata_template.json"
+        filename = "/tmp/import_metadata_template.json"
+
+        strip_double_quotes_for_field(json_file, field, filename)
+        environment = Environment(loader=FileSystemLoader("/tmp/"))
+        template = environment.get_template("import_metadata_template.json")
+
+    content = template.render(
+        version=version,
+        datasource_name=datasource_name,
+    )
+    with open(tmp_json_file, mode="w", encoding="utf-8") as message:
+        message.write(content)
+
+    response = delete_metadata(tmp_json_file)
+    print("delete metadata = ", response.status_code)
+
+    # Import metadata using the specified json
+    response = import_metadata(tmp_json_file)
+    metadata_json = response.json()
+
+    # temporarily moved this up to avoid failures in the subsequent tests
+    response_delete_metadata = delete_metadata(tmp_json_file)
+    print("delete metadata = ", response_delete_metadata.status_code)
+
+    assert response.status_code == int(expected_status_code)
+
+
+@pytest.mark.extended
+@pytest.mark.parametrize("field, expected_status_code, expected_status", mandatory_fields)
+def test_import_metadata_mandatory_fields(cluster_type, field, expected_status_code, expected_status):
+    form_kruize_url(cluster_type)
+
+    # Import metadata using the specified json
+    json_file = "/tmp/import_metadata.json"
+    input_json_file = "../json_files/import_metadata_mandatory.json"
+    json_data = json.load(open(input_json_file))
+
+    if field == 'version':
+        json_data.pop("version", None)
+    else:
+        json_data.pop("datasource_name", None)
+
+    print("\n*****************************************")
+    print(json_data)
+    print("*****************************************\n")
+    data = json.dumps(json_data)
+    with open(json_file, 'w') as file:
+        file.write(data)
+
+    response = delete_metadata(json_file)
+    print("delete metadata = ", response.status_code)
+
+    # Import metadata using the specified json
+    response = import_metadata(json_file)
+    metadata_json = response.json()
+
+    assert response.status_code == expected_status_code, \
+        f"Mandatory field check failed for {field} actual - {response.status_code} expected - {expected_status_code}"
+    assert metadata_json['status'] == expected_status
+
+    response = delete_metadata(json_file)
+    print("delete metadata = ", response.status_code)
+
+
+@pytest.mark.sanity
+def test_duplicate_import_metadata(cluster_type):
```

Comment:
Rename the test name to something appropriate

Reply:
renamed the test to ` test_repeated_metadata_import`